cvat/apps/quality_control/quality_reports.py

```diff
@@ -1972,8 +1979,16 @@ def _find_closest_unmatched_shape(shape: dm.Annotation):
             gt_label_idx = label_id_map[gt_ann.label] if gt_ann else self._UNMATCHED_IDX
             confusion_matrix[ds_label_idx, gt_label_idx] += 1
 
+        if self.settings.match_empty_frames and not gt_item.annotations and not ds_item.annotations:
+            # Add virtual annotations for empty frames
+            valid_labels_count = 1
+            total_labels_count = 1
+
+            valid_shapes_count = 1
+            total_shapes_count = 1
```

Comment:
Shouldn't `ds_shapes_count` and `gt_shapes_count` be updated too?

Reply:
Thanks, updated. I guess the idea was that these are not shapes exactly, but if we add precision and recall in the shape summary section, the metrics will be broken.